presto-hive/src/main/java/com/facebook/presto/hive/HivePartitionManager.java

```diff
@@ -214,7 +212,9 @@ private Table getTable(SemiTransactionalHiveMetastore metastore, SchemaTableName
 
     private List<String> getFilteredPartitionNames(SemiTransactionalHiveMetastore metastore, SchemaTableName tableName, List<HiveColumnHandle> partitionKeys, TupleDomain<ColumnHandle> effectivePredicate)
     {
-        checkArgument(effectivePredicate.getDomains().isPresent());
+        if (effectivePredicate.isNone()) {
+            return ImmutableList.of();
```

Comment:
Why return empty list in this case?

Reply:
There are no partitions.  Since I'm moving https://github.com/prestodb/presto/pull/13069/files/7a57979aa9f71c44c61e6add47d76605a62b4e2a..811947853808aa7aa85ada5151516b8149579eae#diff-66121b6bf6babde202726bd56e496ca7L121 down below where we call `getFilteredPartitionNames`, I need to handle it here.